lib/sources.nix

```diff
@@ -13,7 +13,7 @@ rec {
   # directories, as well as backup files (*~).
   cleanSourceFilter = name: type: let baseName = baseNameOf (toString name); in ! (
     # Filter out Subversion and CVS directories.
```

Comment:
nit: Mind also generalizing this comment?

Reply:
Good catch, did that